packages/select/src/styleSets.ts

```diff
@@ -224,6 +225,22 @@ export const sizeSets: Record<Size, SizeSet> = {
       text: 16,
     },
   },
+  [Size.Fill]: {
```

Comment:
I see a bit of a problem with using the same prop as we use to control the height of the input. I'm also not sure this was the original intent of the ticket. 

If we want to make the input's width more easily controlled, we could consider changing instances of `width` to `min-width`, or removing that altogether. I'd suggest approaching this specific problem the same way we approach TextInput, since the contexts they're used are usually similar.